src/models/artefact.ts

```diff
@@ -25,14 +25,24 @@ export class Artefact {
     constructor(obj: ArtefactDTO) {
         this.id = obj.id;
         this.editionId = obj.editionId;
-        this.imagedObjectId = obj.imagedObjectId;
+        this.imagedObjectId = obj.imagedObjectId || '';
```

Comment:
Maybe `??` is better in this case?  Or maybe the TS/Babel version we use doesn't yet support it.

Reply:
In principal you are right, but we do have a lot of `||` used just for that (because `??` was not introduced into Typescript when we started working on the project), so I don't think it matters much.

Reply:
That is fine too.  If || maintains consistency with the existing code base, that is fine, I will try to keep an eye out for the possible unintended evaluations.